src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPResourceIntTest.kt

```diff
@@ -23,12 +24,15 @@ class CSIPResourceIntTest : IntegrationTestBase() {
   private lateinit var csip1: CSIPReport
   private lateinit var csip2: CSIPReport
   private lateinit var csip3: CSIPReport
+  private lateinit var csipTemplate: IWPTemplate
 
   @BeforeEach
   internal fun createCSIPReports() {
     nomisDataBuilder.build {
+      csipTemplate = template(name = "CSIPA1_FNP", description = "")
```

Comment:
we need a template before we can add a new document - just realised I missed a description - will add this in next PR